frontend/signup.ejs

```diff
@@ -12,4 +12,32 @@
         lunch_dining
         </span>
     <h1 style="margin-left: 5px; margin-top: 0px">HeelMeals</h1>
-</div>
\ No newline at end of file
+</div>
+
+<form action="/login" method="POST">
```

Comment:
The 'Create Account' button is routing to the login page due to this line. If you change this to action='/user/new/' and keep the method="POST", then it'll be calling the correct route in server.js! This route will then create the user, add it to the db, and then redirect the user to a new view page!

Reply:
I see! Thank you for pointing that out. I've corrected that here.